### ISSUE TITLE
Draw with BGRABitmap library

### ISSUE THREAD

**Issue by lainz** [pst2d/csscontrols#3]
Hi, you considered adding more CSS properties with the capabilities of BGRABitmap library?

Like
- gradients
- antialiased borders
- transparency

**Comment by lainz** [comment]
BTW There is BGRABitmap for Delphi as well, if that is important to you.
https://github.com/bgrabitmap/BGRABitmapDelphi

**Comment by pst2d** [comment]
yes. I'm working now on support box-shadow, more background prop. (like gradients). For box-shadow advanced clipping is required. On CSS side Weiler–Atherton will be enough. https://en.wikipedia.org/wiki/Weiler%E2%80%93Atherton_clipping_algorithm (there is no self-intersecting polygons) but I'm not sure about aliasing precision with this solution.

**Comment by lainz** [comment]
Ok, thanks. Glad to know that is being active developed.

**Comment by lainz** [comment]
I'm not the author of BGRABitmap, just a contributor and I help making BGRAControls. But if you need help with antialiasing, maybe ask @circular17 in the bgrabitmap repository or in the forum as circular, he is really kind and he maybe can help you.

**Comment by CodingMadness** [comment]
Hey guys is this entire Project still alive and actively developed, since I saw the last commits are ~8months old, I really hope this genius project is still maintained :)

**Comment by lainz** [comment]
Last commit in the dev branch is 8 months old.

**Comment by pst2d** [comment]
Please be patient i will work on it :) ASAP (jan-feb.2021).

**Comment by lainz** [comment]
Thanks. If you have time try bgrabitmap.
We did some svg controls. Maybe you can add svg support.

**Comment by formlessdao** [comment]
So...why did you give up pst2d ?
